File: Compiler/UnitTestSample_1/NormalClass3.cs
using System;

namespace UnitTestSample_1
{
    class NormalClass3
    {
        private int _counter;
        public void IncreaseBy(int x)
        {
            _counter += x;
        }

        public void Increase()
        {
            _counter++;
        }

        public void Reset()
        {
            _counter = 0;
        }
    }
}

File: Compiler/UnitTestSample_1/ToCompare/NormalClass3.js
enyo.kind({
    name: "UnitTestSample_1.NormalClass3",
    kind: "enyo.Object",

    _counter : 0,
    increaseBy : function(x){
        this._counter += x;
    },

    increase : function(){
        this._counter++;
    },

    reset : function(){
        this._counter = 0;
    }
});
